documentation/misc/Building_a_network_skeleton.md

```diff
@@ -144,11 +173,17 @@ You can take a look at [this PR as a reference](https://github.com/filecoin-proj
 2. Patch the FVM-dependency (fvm4 and fvm4_shared) in `rust/cargo.toml` to use the newly published Ref-FVM release.
     -  Add `features = ["nvXX+1-dev"]`.
 
-You can take a look at this [Filecoin-FFI PR as a reference](https://github.com/filecoin-project/filecoin-ffi/pull/454), which added the skeleton for network version 23.
+    👉 You can take a look at this [Filecoin-FFI PR as a reference](https://github.com/filecoin-project/filecoin-ffi/pull/454), which added the skeleton for network version 23.
+
+3.  [Follow the release process](https://github.com/filecoin-project/filecoin-ffi/blob/master/RELEASE.md) to publish `v1.NEW_LOTUS_MINOR_VERSION.0-dev`
```

Comment:
@rjan90 : I assume we need to release filecion-ffi (like you did for nv24 in https://github.com/filecoin-project/filecoin-ffi/pull/481 I think)